src/components/molecules/PanelOptions/PanelOptions.tsx

```diff
@@ -16,9 +16,9 @@ export const PanelOptions = (props: PanelOptionsProps) => (
         <BoxButton
           className={'Panel PanelOptions__box-buttons__CreateToken'}
           image={Images.KeySmall}
-          title={'API Keys'}
+          title={'API Tokens'}
           description={'Get started building out an integration on top of po.et using the frost API.'}
-          buttonText={'Get API Key'}
```

Comment:
Just a small detail but should this one say Get Api Token(s)? Since the button doesn't actually create a token?

Reply:
Good question. It really could go either way. I think you're right though because the "create" part does not happen until the next screen. I'll change it.